### PR TITLE
Can have multiple panels in the application

### PR DESCRIPTION
**Pull request by samikuikka**
Now user can have multiple panels in the application.

User adds panels from the navigation drawer.

Panels have persistent state between navigation

### CODE REVIEW

File: frontend/src/components/NavigationContainer/NavigationDrawer/PanelSection/index.tsx [hunk 1]
```diff
@@ -3,20 +3,55 @@ import { NavigationSubHeader } from '../NavigationSubHeader';
...
+                    return (
+                        <NavigationLink
+                            key={panelId}
+                            label={`Panel-${index + 1}`}
+                            to={`/panels/${panelId}`}
+                            icon={solidIcon('CubeIcon')}
+                        />
+                    );
+                }
+            })}
```
Comment (itulkki) [inline frontend/src/components/NavigationContainer/NavigationDrawer/PanelSection/index.tsx:54]: Could these lines from 34 to 54 be simplified or moved to a function outside the component?

File: frontend/src/components/ContentPanel/index.tsx [hunk 2]
```diff
@@ -49,25 +49,64 @@ export const ContentPanel: FC<ContentPanelProps> = ({
         setPromptBoxes((prev) => [...prev, newBox]);
     };
 
-    //Callbacks to asynchronously fetch AI data from backend
-    const generateAll = () => {
-        promptBoxes.forEach((p) => {
+    // Generates all the IO boxes that are not locked
```
Comment (itulkki) [inline frontend/src/components/ContentPanel/index.tsx]: This component is getting pretty gigantic. Maybe some of the stuff there could be moved elsewhere.

Comment (samikuikka) [reply frontend/src/components/ContentPanel/index.tsx]: I'll move the logic to the custom hook 👍

**Review by itulkki — APPROVED** [review]

**Review by KorpelainenTommi — APPROVED** [review]
:+1:

**Pull request merged**